code/modules/vore/eating/bellymodes_vr.dm

```diff
@@ -1,17 +1,13 @@
 // Process the predator's effects upon the contents of its belly (i.e digestion/transformation etc)
-/obj/belly/proc/process_belly(times_fired, wait) //Passed by controller
-	if((times_fired < next_process) || !contents.len)
-		recent_sound = FALSE
-		return SSBELLIES_IGNORED
+/obj/belly/process(wait) //Passed by controller
+	recent_sound = FALSE
 
 	if(loc != owner)
 		if(istype(owner))
 			loc = owner
 		else
 			qdel(src)
-			return SSBELLIES_PROCESSED
-
-	next_process = times_fired + (6 SECONDS / wait) //Set up our next process time.
+			return
```

Comment:
```suggestion
			return PROCESS_KILL
```
It just deleted itself, so removing from list is good?  Although... I guess Destroy() will handle it.

Reply:
Hm! I suppose. Though it will do the work twice, yes. No point since STOP in Destroy().